dummyserver/server.py

```diff
@@ -16,6 +16,11 @@
 import warnings
 from datetime import datetime
 
+try:
+    from ssl import CERT_OPTIONAL
+except:
+    CERT_OPTIONAL = None
```

Comment:
I can confirm that `ssl.CERT_OPTIONAL` is present in py26+.  I'm not sure why @theonewolf put the defensive coding around the import, but I'm fairly certain it doesn't need to be there.  In other tests dealing with SSL, I don't see any protections around `import ssl` statements, so I think this can safely be refactored back down to a simple from/import.